<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncTab</class>
 <widget class="QWidget" name="SyncTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="sourceGB">
     <property name="title">
      <string>Square wave source (set period=1s, duty cycle=50%)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>101</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>    Measured period</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="sourceCB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Disable sync waveform</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Separate high precision pulser</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Current NI acquisition device</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="sourceSB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>2.500000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sourceLE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="calGB">
     <property name="title">
      <string>Calibration run: Measure and update sample rates</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="calChk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="text">
         <string>Use next run for calibration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>    Data collection time (min)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="minSB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="chanGB">
     <property name="title">
      <string>Inputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="2" column="2">
       <widget class="QSpinBox" name="niChanSB">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Imec</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nidq</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="niChanCB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Digital bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Analog channel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>PXI SMA on slot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="imSlotSB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Note: Connect one PXI SMA only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Threshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="niThreshSB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Imec OneBoxes: Connect sync in to each 'SMA 1' (if not source) </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceCB</tabstop>
  <tabstop>sourceSB</tabstop>
  <tabstop>imSlotSB</tabstop>
  <tabstop>niChanCB</tabstop>
  <tabstop>niChanSB</tabstop>
  <tabstop>niThreshSB</tabstop>
  <tabstop>calChk</tabstop>
  <tabstop>minSB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
